src/test/regress/sql/pg14.sql

```diff
@@ -79,5 +79,49 @@ SELECT result AS column_compression FROM run_command_on_workers($$SELECT ARRAY(
 SELECT attname || ' ' || attcompression FROM pg_attribute WHERE attrelid::regclass::text LIKE 'pg14.col\_compression%' AND attnum > 0 ORDER BY 1
 )$$) ORDER BY length(result);
 
+RESET citus.multi_shard_modify_mode;
+
+-- test procedure OUT parameters with procedure pushdown
+CREATE TABLE test_proc_table (a int);
+
+create or replace procedure proc_pushdown(dist_key integer, OUT created int4[], OUT res_out text)
```

Comment:
I was curious what happens if we only have IN parameters and then try to add an OUT parameter but it seems that it is not allowed:

```
ERROR:  cannot change whether a procedure has output parameters
```

So the current code seems quite safe